### DISCUSSION TITLE
Have a GitHub bot that adds a comment saying in which release a pull request landed

### DISCUSSION THREAD

**Discussion by 0xced** [dotnet/runtime#47939] [Ideas] [answered]
This is a problem I have encountered [several](https://github.com/dotnet/corefx/pull/28516#issuecomment-444304388) [times](https://github.com/Azure/azure-functions-vs-build-sdk/pull/140#issuecomment-355019345) across different dotnet repositories. You see a pull request have been merged but you have no idea in which exact release it landed.

The fastlane repository does this and it's awesome, see https://github.com/fastlane/fastlane/pull/17611#issuecomment-730589277 for example.

**Comment by danmoseley** [comment]
@mmitche has this been considered?

**Comment by jkotas** [comment, marked as answer]
You can find the list of releases that the PR/commit is from in the github UI. For example, go to https://github.com/dotnet/runtime/pull/40637 and click on the " jkotas merged commit 7d152b7". You can see list of all releases that the change is in:

![image](https://user-images.githubusercontent.com/6668460/107155263-5629f580-692c-11eb-90b9-fa9cb39e3ad1.png)

You are right that there is no way to get actively notified today that would be achieved by adding a comment. Adding comment to the PR once it is included in a release will result in a massive spam for the people actively working on the repo each time we snap a release that is likely to have bad side-effects like spam filters blocking all github email, etc. I can see that it may ok for smaller repos with lower traffic (dotnet/runtime has ~20x more PR traffic than fastlane from your example).

**Reply by vcsjones** [reply]
A possible alternative to noisy comments: would be nice if PRs consistently had the milestone assigned. This seems to happen sometimes. It appears this is already done for servicing PRs.

**Reply by 0xced** [reply]
Excellent, thanks for teaching us how to use the GitHub UI to get this much needed information!

I think it would be nice to have a comment to be actively notified but I totally understand your concern about the high pull request volume on such a big repository.
